c7n/resources/swf.py

```diff
@@ -0,0 +1,65 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.utils import local_session
+from c7n import tags
+
+
+@resources.register('swf')
+class SimpleWorkflowDomain(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = arn_type = 'swf'
```

Comment:
the arns for these are a little interesting, afaics type is an empty string as it uses a resource prefix of  'domain/' instead.